pages/company/components/Story.tsx

```diff
@@ -0,0 +1,83 @@
+/* eslint-disable react/no-unescaped-entities */
+import { Typography } from "@mui/material";
+import { Box } from "@mui/system";
+import { FC } from "react";
+import styles from "../company.module.scss";
+import { story4 } from "assets/png";
+import Image from "next/image";
+
+const story = [
+	{ title: "	✓ Founded in 2014" },
+	{ title: "	✓ Community of 125K+" },
+	{ title: "	✓ $33M in funding" },
+	{ title: "	✓ 500+ global meetups" },
+];
+export const Story: FC = () => {
+	return (
+		<div className={styles.story}>
+			<Typography
+				variant="h4"
+				sx={theme => ({
+					[theme.breakpoints.down("sm")]: {
```

Comment:
change breakpoint to center the text in tablet made

Reply:
better use `sx={ { md : {textAlign: "center"} } }` ==> this one looks short and simpler